app/commands/user/premium/expire.rb

```diff
@@ -0,0 +1,7 @@
+class User::Premium::Expire
+  include Mandate
+
+  initialize_with :user
+
+  def call = User::Notification::CreateEmailOnly.defer(user, :expired_premium)
```

Comment:
Do we call this anywhere yet? I'm guessing not?

Reply:
We don't yet. We do now :)